Overview.src.html

```diff
@@ -688,25 +705,25 @@ <h4 id=terminology-headers>Headers</h4>
   <tr>
    <td>"<code title>fetch</code>"
    <td>""
-   <td>"<code title>subresource</code>"
+   <td>"<code title>fetch</code>"
    <td><code title>connect-src</code>
-   <td><code title=dom-global-fetch>fetch()</code>, Cache API?
+   <td><code title=dom-global-fetch>fetch()</code>, <code>XMLHttpRequest</code>, Cache API?
```

Comment:
The only thing that's different is that "fetch" does not get an Accept-Language header by default. We did this to give developers more control. Originally we also did that for Accept, but apparently that broke too many things.